src/main/scala/cromwell/binding/values/WdlMap.scala

```diff
@@ -75,4 +79,12 @@ case class WdlMap(wdlType: WdlMapType, value: Map[WdlValue, WdlValue]) extends W
     }
     collected.flatten.toSeq
   }
+
+  override def getHash(implicit hasher: FileHasher): SymbolHash = {
+    val hashedMap = value map {
+      case (k, v) => k.getHash -> v.getHash
+    }
```

Comment:
Is this guaranteeing the order of `hashedMap`?  I am new to this `Hlist` concept.